api/maketdata.go

```diff
@@ -167,10 +174,13 @@ func makeChartsRoute(router gin.IRouter) {
 // @Param time_start query int false "Start timestamp" default(1574483028)
 // @Param currency query string false "The currency to show coin info in" default(USD)
 // @Success 200 {object} blockatlas.ChartCoinInfo
-// @Router /v1/market/info [get]
-func makeCoinInfoRoute(router gin.IRouter) {
+// @Router /market/v1/info [get]
```

Comment:
is this intended change too?

Reply:
no, the old routes were wrong in the swagger file and I made a confusion, fixed now